src/transformers/trainer.py

```diff
@@ -1380,6 +1390,11 @@ def _wrap_model(self, model, training=True, dataloader=None):
                     size_based_auto_wrap_policy,
                     transformer_auto_wrap_policy,
                 )
+
+                if self.is_fsdp_xla_v2_enabled:
+                    from torch_xla.experimental.spmd_fully_sharded_data_parallel import (
+                        SpmdFullyShardedDataParallel as FSDPv2,
```

Comment:
Could we make this easier by importing `FSDPv2` as `FSDP` instead?

Reply:
May I ask what's the benefits of doing so?